templates/zerver/login.html

```diff
@@ -132,9 +132,9 @@ <h1 class="get-started">{{ _("Log in to Zulip") }}</h1>
 
                         {% for backend in social_backends %}
                         <div class="login-social">
-                            <form class="social_login_form form-inline {{ backend.name }}-wrapper" action="{{ backend.login_url }}" method="get">
+                            <form class="social_login_form form-inline" action="{{ backend.login_url }}" method="get">
                                 <input type="hidden" name="next" value="{{ next }}">
-                                <button class="login-social-button">
+                                <button class="login-social-button" style="background-image:url({{ backend.display_logo }})">
                                     {{ _('Log in with %(identity_provider)s', identity_provider=backend.display_name) }}
```

Comment:
We need these changes in `accounts_home.html` as well; otherwise /register will have broken buttons.

Reply:
Ouch, I missed that somehow - fixed now in that commit.